sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpIniSensor.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.php;
+
+import java.util.List;
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.rule.CheckFactory;
+import org.sonar.api.batch.rule.Checks;
+import org.sonar.api.batch.sensor.Sensor;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.SensorDescriptor;
+import org.sonar.api.batch.sensor.issue.NewIssue;
+import org.sonar.api.batch.sensor.issue.NewIssueLocation;
+import org.sonar.api.rule.RuleKey;
+import org.sonar.php.checks.CheckList;
+import org.sonar.php.ini.PhpIniCheck;
+import org.sonar.php.ini.PhpIniIssue;
+import org.sonar.php.ini.PhpIniParser;
+import org.sonar.php.ini.tree.PhpIniFile;
+
+public class PhpIniSensor implements Sensor {
+
+  private final CheckFactory checkFactory;
+
+  public PhpIniSensor(CheckFactory checkFactory) {
+    this.checkFactory = checkFactory;
+  }
+
+  @Override
+  public void describe(SensorDescriptor descriptor) {
+    descriptor.name("php.ini analyzer");
```

Comment:
@pynicolas `Analyzer for "php.ini" files`? (IMHO it's better to have name from capital letter)